.factorypath

```diff
@@ -0,0 +1,212 @@
+<factorypath>
```

Comment:
was this file intended to be checked in?

Reply:
Yes this was unintended. Have removed it.